defaults/main.yml

```diff
@@ -14,6 +14,7 @@ postgresql_default_auth_method: "trust"
 # The user/group that will run postgresql process or service
 postgresql_service_user: "{{ postgresql_admin_user }}"
 postgresql_service_group: "{{ postgresql_admin_user }}"
+postgresql_service_enabled: false
```

Comment:
Because it was disabled by default. I decided not to break behaviour. Do you think it's reasonable to enable by default?